.travis.yml

```diff
@@ -3,6 +3,7 @@ git:
   depth: 9999999
 jdk:
   - oraclejdk8
+  - oraclejdk13
```

Comment:
is it allowed to have multiple versions here?

Reply:
yes, and there would be a build for each